src/HttpServer/HttpServer.cc

```diff
@@ -24,7 +24,13 @@ using json = nlohmann::json;
 
 namespace carta {
 
-const std::string success_string = json({{"success", true}}).dump();
+const std::string SUCCESS_STRING = json({{"success", true}}).dump();
```

Comment:
This is unnecessary and should be reverted. While this is repeated across multiple functions, we don't abstract away other kinds of responses, and it's more readable to keep this directly in the functions.